source/blog/2018-03-26-ember-3-1-released.md

```diff
@@ -0,0 +1,149 @@
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.1
+
+Ember.js 3.1 is an incremental, backwards compatible release of Ember with
+bugfixes, performance improvements, and minor deprecations.
+
+#### Deprecations in Ember 3.1
+
+Deprecations are added to Ember.js when an API will be removed at a later date.
+
+Each deprecation has an entry in the deprecation guide describing the migration
+path to more stable API. Deprecated public APIs are not removed until a major
+release of the framework.
+
+Consider using the
+[ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow)
+addon if you would like to upgrade your application without immediately addressing
+deprecations.
+
+Two new deprecations are introduces in Ember.js 3.1:
+
+* TODO
+* TODO
+
+For more details on changes in Ember.js 3.1, please review the
+[Ember.js 3.1.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.1.0).
+
+### Upcoming Changes in Ember.js 3.2
+
+Ember.js 3.2 will introduce two new features:
+
+* TODO
+* TODO
+
+#### Deprecations in Ember.js 3.2
+
+Two new deprecations are introduces in Ember.js 3.2:
+
+* TODO
+* TODO
+
+For more details on the upcoming changes in Ember.js 3.2, please review the
+[Ember.js 3.2.0-beta.1 release page](https://github.com/emberjs/ember.js/releases/tag/v3.2.0-beta.1).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.1
+
+#### Deprecations in Ember Data 3.1
+
+Two new deprecations are introduces in Ember Data 3.1:
+
+* TODO
+* TODO
+
+For more details on changes in Ember Data 3.1, please review the
+[Ember Data 3.1.0 release page](https://github.com/emberjs/data/releases/tag/v3.1.0).
+
+
+### Upcoming changes in Ember Data 3.2
+
+
+#### Deprecations in Ember Data 3.2
+
+For more details on the upcoming changes in Ember Data 3.2, please review the
+[Ember Data 3.2.0-beta.1 release page](https://github.com/emberjs/data/releases/tag/v3.2.0-beta.1).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js
+applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI separately from Ember.js and Ember Data! To upgrade
+your projects using `yarn` run:
+
+```
+yarn upgrade ember-cli
+```
+
+To upgrade your projects using `npm` run:
+
+```
+npm install --save-dev ember-cli
+```
+
+After running the
+upgrade command run `ember init` inside of the project directory to apply the
+blueprint changes. You can preview those changes for [applications](https://github.com/ember-cli/ember-new-output/compare/v3.0.0...v3.1.0)
+and [addons](https://github.com/ember-cli/ember-addon-output/compare/v3.0.0...v3.1.0).
+
+### Changes in Ember CLI 3.1
```

Comment:
- [#7601](https://github.com/ember-cli/ember-cli/pull/7601) blueprints/addon: Replace `test` with `test:all` [@Turbo87](https://github.com/Turbo87)

`test` will run `ember test` now (like in apps), `test:all` will use `ember-try` to run tests in all configured scenarios

- [#7492](https://github.com/ember-cli/ember-cli/pull/7492) Use yarn instead of npm when part of a yarn workspace root [@thetimothyp](https://github.com/thetimothyp)

`yarn.lock` file detection improved

- [#7573](https://github.com/ember-cli/ember-cli/pull/7573) Install optional dependencies when creating a new project [@t-sauer](https://github.com/t-sauer)

Fixes issues with the Glimmer blueprints

- [#7586](https://github.com/ember-cli/ember-cli/pull/7586) add feature flag to project.isModuleUnification [@GavinJoyce](https://github.com/GavinJoyce)
- [#7541](https://github.com/ember-cli/ember-cli/pull/7541) Add `project.isModuleUnification()` [@GavinJoyce](https://github.com/GavinJoyce)

Module Unification continues...